src/EnergyPlus/CMakeLists.txt

```diff
@@ -826,21 +829,6 @@ install( TARGETS energyplusapi
   ARCHIVE DESTINATION ./
 )
 
-# we will build the intermediate wrapper DLL that hard links to Python DLL here
```

Comment:
Cleaning out the little Python wrapper blob from the build process and runtime process is the second important piece of this.